jsonrpc/src/main.ts

```diff
@@ -1090,6 +1124,12 @@ function _createMessageConnection(messageReader: MessageReader, messageWriter: M
 			});
```

Comment:
We could here when the promise resolves cleanup the cancellation file if it got created.

Reply:
ah, you already pointed out this code. so rather than "then", just put the code in the line 1123?